src/index.html

```diff
@@ -11,7 +11,215 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      id="form"
+    >
+      <fieldset class="fieldSet">
+        <legend>Personal information</legend>
+
+        <div>
+          <label class="form_field" for="sname">Surname:</label>
+          <input
+            class="form_element"
+            type="text"
+            name="surname"
+            id="sname"
+          >
+          <br>
+        </div>
+
+        <div>
+          <label class="form_field" for="name">Name:</label>
+          <input
+            class="form_element"
+            type="text"
+            name="name"
+            id="name"
+          >
+          <br>
+        </div>
+
+        <div>
+          <label class="form_field" for="age">How old are You?</label>
+          <input
+            class="form_element"
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+            id="age"
+          >
+          <br>
+        </div>
+
+        <div>
+          <label class="form_field" for="dobirth">Full date of birth:</label>
+          <input
+            class="form_element"
+            type="date"
+            id="dobith"
+            name="dobirth"
+            value="yyyy-mm-dd"
+          >
+          <br>
+
+        </div>
+
+        <div>
+          <label class="form_field" for="termAgreement">I accept the term of the agreement</label>
+          <input
+            class="checkbox"
+            type="checkbox"
+            id="termAgreement"
+            name="termAgreement"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldSet">
+        <legend>Registration</legend>
+
+        <label class="form_field" for="email">E-mail:</label>
+        <input
+          class="form_element"
+          type="email"
+          name="email"
+          id="email"
+          placeholder="email@example.com"
+          minlength="6"
+          maxlength="30"
+          required
+          multiple
+        >
+        <br>
+
+        <label class="form_field" for="password">Password:</label>
+        <input
+          type="password"
+          name="password"
+          id="password"
+          minlength="6"
+          maxlength="20"
+          required
+        >
+
+      </fieldset>
```

Comment:
```suggestion
        >
      </fieldset>
```
Don't leave newlines between parent and child tags